docs/sphinx/omero-pyramid/index.rst

```diff
@@ -0,0 +1,22 @@
+Internal OMERO pyramid format v1.0.0
+====================================
+
+For files that contain very large images and are not in a format that supports pyramids, OMERO will generate its own
+image pyramid to improve visualization performance.  Bio-Formats can read these generated pyramids, but cannot
+currently write them outside of OMERO.  For details of how to read image pyramids with Bio-Formats, see :bf_doc:`Working with whole slide images <developers/wsi.html>`
+
+The OMERO pyramid format is a :bf_doc:`TIFF <formats/tiff.html>` file containing JPEG-2000 compressed image tiles.  All resolutions for a tile
+are encoded in the same JPEG-2000 stream, using the "decompression levels" feature of JPEG-2000.
+As a result, only data types supported by the JPEG-2000 standard (``uint8`` and ``uint16``) are supported.
+Images with pixel type ``uint32``, ``float`` (32-bit floating point), or ``double`` (64-bit floating point) cannot be converted to
+an OMERO pyramid.  Pyramid files larger than 4 gigabytes are supported, as are pyramids containing multiple channels,
+Z sections, and/or timepoints.
+
+Each pyramid contains 5 resolutions for each image plane, with each resolution stored in descending order from largest to smallest XY size.
+Each resolution is half the width and height of the previous resolution.
```

Comment:
It might be worth mentioning explicitly that these 5 resolutions are separate IFDs.

Also, it made me wonder what the purpose of the JPEG-2000 decompression levels is, if we store and use the separate IFDs and don't actually use the decompression levels in the full-resolution IFD?